articles/fuzzy-logic-operations/index.md

```diff
@@ -0,0 +1,138 @@
+
+#### L- function
+This function is the inverse of $\gamma$ function. It is given as follows:
+
+$$ L(x,\alpha,\beta) = \begin{cases} 
+1 & x \le \alpha \\
+\frac{\alpha - x}{\beta - x} & \alpha \le x \le \beta \\ 
+0 & x \gt \beta 
+\end{cases} $$
+ 
+![L-function](L-function.png)
+#### Triangular function
+
+$$ L(x,\alpha,\beta,\gamma) = \begin{cases} 
+0 & x\le \alpha \\
+\frac{x-\alpha}{\beta - \alpha} & \alpha \le x \le \beta \ \\
+\frac{\alpha - x}{\beta - \alpha} & \beta \le x \le \gamma \\
+0 & x \gt \gamma
+\end{cases}$$
+
+![triangular function](triangular_function.png)
+
+#### pi function
+
+This function gets its name from the shape of pi. This function looks similar to the symbol pi ($\pi$). Mathematically it is given as follows:
+
+$$ \pi(x, \alpha, \beta,\gamma, \delta) = \begin{cases} 
+0 & x\le \alpha \\
+\frac{x - \alpha }{\beta - \alpha} & \alpha \lt x \le \beta \ \\
+1 & \beta \lt x \le \gamma \\
+\frac{\gamma - x }{\delta - \gamma} & \gamma \lt x \le \delta \
+\end{cases}$$
+
+![pi-function](pi_function.png)
+
+#### Gaussian function
+
+Derived from the normal distribution, this function works in most scenarios, where information about the data is unknown. In cases dealing with sampled data, this function is used extensively.
+
+Mathematically, it is given as follows:
+
+$$ 
+G(x,\mu,\sigma) = exp (-\frac{(x-\mu)^2}{2\sigma^2})
+$$
+
+### Operations in fuzzy logic
+
+Various operations can be performed on a crisp set. The same set of operations can be performed on fuzzy sets, with minor modifications. The operations allow changes to be made to the sets. These operations come in handy while building the applications. 
+
+### Types of operations in fuzzy logic
+
+#### Union 
+
+Union operation concatenates the two sets and produces a new set. Since the fuzzy sets may have the same element with varying degrees of membership, the union operation considers the element once. The degree of membership for this value is the maximum value the element has. Hence, this can be viewed as finding the elements with a maximum degree of membership.
+
+#### Intersection
+
+Intersection operation finds the common elements between the two sets. In this case, if an element exists in both sets, the element with the least degree of membership is retained in the new set. The intuition behind the intersection operation is to find the common elements between the two sets. Therefore, the element with the lowest degree of membership exists in both the sets. This operation is also viewed as finding the elements in both sets with the minimum degree of membership.
+
+#### Complementation
+
+The degree of membership for a fuzzy set can range between 0 & 1. Given a set of elements with a degree of memberships, the fuzzy set's complement will behave inverted degrees of membership. The inverted degree of membership for an element x is defined as follows:
+
+$$new\; degree\; of\; membership(x)\; = 1\; -\; degree\; of\; membership(x)$$
+
```

Comment:
Could you please change the equation to render without these semi-colons?

![image](https://user-images.githubusercontent.com/34317373/105633551-4071f600-5e7f-11eb-833e-cd147e57b3d3.png)

Reply:
![image](https://user-images.githubusercontent.com/26842636/105848869-996b9680-6005-11eb-9766-b3db11d3b037.png)
Again, those semi colons are responsible for spacing. @hectorkambow Your assistance is needed here as well. Thank you :)